### ISSUE TITLE
Update translations

### ISSUE THREAD

**Issue by benbucksch** [mustang-im/mustang#310]
Tasks (in this order):

### Update from weblate
- [x] Go to [Weblate](https://hosted.weblate.org/projects/mustang/mustang/) and check the [translation changes by translators](https://hosted.weblate.org/projects/mustang/mustang/#history) there. See section Attention below.
- [x] Merge all translation changes from Weblate into a branch, and create a PR.
- [x] Check the output, and fix any problems you find.
- [x] Tell me when it's ready to merge.

### Translate new strings
- [ ] Start with the branch from Weblate above.
- [x] Extract new strings
- [ ] Run AI translators
- [ ] Make a PR
- [ ] Check the output, and fix any problems you find. See section Attention below.
- [ ] Tell me when it's ready to merge.

### Test
(with both Weblate changes and AI translator changes)
- [ ] Test it, in all languages.
- [ ] Fix any problems you find. Repeat previous step until there are no more issues.

### Attention
Pay particular attention to
* Any placeholders `{}` and plural forms, that the translator doesn't mess this up.
* Folder properties (select folder, click [...], check the message counts)
* Settings (all panels), as they contain lots of strings.

**Comment by jermy-c** [comment]
After doing some research, it seems like Weblate is prone to merge conflicts when ever the Weblate repo is out of sync with the upstream repo. This would happen even if you don't do a force push.

Why is this a problem? Because to fix it, you would lose some translations in the process:

1. Reset all changes in the Weblate repository: you would lose the translations by the Translators
2. Force writing all translations to the Weblate repository: you would lose the translations by the Translators
3. Rebasing the upstream repository with Weblate changes: you would lose the translations by Auto Translate

And you can't resolve the conflicts:

- Remotely: Weblate doesn't offer a client or API to do that, and 
- Locally: There's no way to push to the Weblate repo after resolving the conflicts locally

**Comment by jermy-c** [comment]
For some reason the first load of the Setup Dialog doesn't display the translation correctly. On dev and builds after install.

<img width="598" alt="Image" src="https://github.com/user-attachments/assets/19b36baa-2cc0-4969-b4ac-e0d826372cce" />

**Comment by benbucksch** [comment]
I also see the "S" in mail, as title of the msg list, when I do a search.

**Comment by benbucksch** [comment]
Indeed, that was the reason. Fixed in git commit ec5a9291. Thanks, Jeremy!
